src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -178,14 +178,17 @@ export const getBookInformation = async(
 
   } else if (UNLIMITED_CONTENT) {
     for (const {id, bookVersion} of allReferences) {
+      // Omit references that don't have bookVersion because new pipeline will not support requests without a version
+      if (!bookVersion) { continue; }
+
       const osWebBook =  await services.osWebLoader.getBookFromId(id).catch(() => undefined);
       const archiveBook = await services.archiveLoader.book(id, bookVersion).load();
       if (archiveBook && archiveTreeSectionIsBook(archiveBook.tree)) {
         return {osWebBook, archiveBook};
       }
     }
   }
-  return undefined;
+  return new ReferenceLoadingError();
```

Comment:
should this throw instead of return?

actually i feel like it would have been easier to just leave this as `undefined`